Makefile

```diff
@@ -5,11 +5,16 @@ export
 STEPS := all tests apps clean check info tools %.debug
 .PHONY: $(STEPS) %.debug shell docker
 
+NPROC := nproc
```

Comment:
I get `make: nproc: Command not found` when run on mac due to this line. However, the make command succeeds as it goes to the darwin step below and find the right nproc but still not great to see the error. I would suggest doing the following instead:

```
NPROC =
ifeq ($(HOST),Darwin)
  NPROC = sysctl -n hw.physicalcpu
else
  NPROC = nproc
endif
```

Reply:
Done.